perception/conda/build_conda_pkg.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -xe
+
+conda build --no-anaconda-upload .
+
+conda install anaconda-client
+#export ANACONDA_PASSWORD="5T!%NUjCZy529@d"
```

Comment:
remove these lines, and probably change your password immediately?
Instead, read environment variables